helpers/GenerateResourcesAndImage.ps1

```diff
@@ -240,13 +240,19 @@ Function GenerateResourcesAndImage {
 
     try {
         # Login to Azure subscription
+        try {
+            $azAccount = az account show -o none
+            Write-Warning "Already logged in..."
```

Comment:
Why is it warning? It looks like expected behaviour for me. Should it be `Write-Verbose`?